config/environments/development.rb

```diff
@@ -49,19 +54,26 @@
   # number of complex assets.
   config.assets.debug = true
 
-  # Supress logger output for asset requests
+  # Suppress logger output for asset requests.
   config.assets.quiet = true
 
-  # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
+  # Raises error for missing translations.
+  # config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
-  # In development, we want any emails to be opened in the browser
+  # In development, we want any emails to be opened in the browser.
   config.action_mailer.delivery_method = :letter_opener
 
+  # Uncomment if you wish to allow Action Cable access from any origin.
+  # config.action_cable.disable_request_forgery_protection = true
+  config.action_cable.url = "wss://localhost:3000/cable"
```

Comment:
where is this coming from?

I'm concerned about the `-p` param to change port number and this being hard-coded.

Reply:
I've gone back through the steps that lead me to this and I think it was a db/page reload red-herring. I believe i've reproduced it, and this just isn't needed. Will remove.